sources/packages/backend/apps/api/src/route-controllers/student/student.controller.service.ts

```diff
@@ -64,25 +71,57 @@ export class StudentControllerService {
     groupName: string,
     auditUserId: number,
   ): Promise<FileCreateAPIOutDTO> {
-    const createdFile = await this.fileService.createFile(
-      {
-        fileName: file.originalname,
-        uniqueFileName: uniqueFileName,
-        groupName: groupName,
-        mimeType: file.mimetype,
-        fileContent: file.buffer,
-      },
-      studentId,
-      auditUserId,
-    );
-
-    return {
-      fileName: createdFile.fileName,
-      uniqueFileName: createdFile.uniqueFileName,
-      url: `student/files/${createdFile.uniqueFileName}`,
-      size: createdFile.fileContent.length,
-      mimetype: createdFile.mimeType,
-    };
+    let uploadStatus: number;
+    try {
+      this.logger.log(`Uploading file ${file.originalname} to S3 storage.`);
+      uploadStatus = await this.objectStorageService.putObject({
+        key: uniqueFileName,
+        contentType: file.mimetype,
+        body: file.buffer,
+      });
+    } catch (error: unknown) {
+      this.logger.error(parseJSONError(error));
```

Comment:
In case an error happens during the `putObject` the `FILE_UPLOAD_SERVICE_UNAVAILABLE` exception can be thrown from here and abort the rest of the method execution.